CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Added support for Kafka Exporter `offset.show-all` parameter
 * Prevent removal of the `broker` process role from KRaft mixed-nodes that have assigned partition-replicas
 * Improve broker scale-down prevention to continue in reconciliation when scale-down cannot be executed
+* Added topic replication factor change support when Cruise Control integration is enabled
```

Comment:
If this is really merged for 0.40, it should make it more clear that this is UTO only functionality.

Reply:
Changed mentioning UTO.